samples/bluetooth/df_per_adv_beacon/sample.yaml

```
@@ -0,0 +1,7 @@
+sample:
+  name: Direction Finding Connectionless Beacon
+tests:
+  sample.bluetooth.df_beacon:
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
+    tags: bluetooth
```

Could you add the `AoA` only mode to this, to make sure it's included when running twister:
```suggestion
    tags: bluetooth
  sample.bluetooth.df_beacon.aoa:
    extra_args: OVERLAY_CONFIG="overlay-aoa.conf" DTC_OVERLAY_FILE="boards/nrf52833dk_nrf52833_aoa_antenna.overlay"
    platform_allow: nrf52833dk_nrf52833
```

---

Added second configuration (`sample.bluetooth.df_beacon.aoa`) for twister build.